demos/dropbase.ftd

```diff
@@ -144,7 +144,7 @@ width: fill-container
 -- component footer:
 
 -- ds.section-column: asdas
-inner-background.solid: $ds.colors.background.step-2
+;;inner-background.solid: $ds.colors.background.step-2
```

Comment:
Why did you do this?

Reply:
This was done to test if the background image is working or not, will fix this.